OpenRA.Mods.Common/Effects/Bullet.cs

```diff
@@ -164,8 +165,11 @@ public void Tick(World world)
 			if (info.ContrailLength > 0)
 				contrail.Update(pos);
 
-			if (ticks++ >= length || (info.Blockable && world.ActorMap
-				.GetUnitsAt(world.Map.CellContaining(pos)).Any(a => a.HasTrait<IBlocksProjectiles>())))
+			if (ticks++ >= length
```

Comment:
You'll have to check two times per tick now. The current location and the next location.